frontend_app_workspace_advanced/src/container/WorkspaceAdvanced.jsx

```diff
@@ -54,25 +54,25 @@ class WorkspaceAdvanced extends React.Component {
     addAllResourceI18n(i18n, this.state.config.translation, this.state.loggedUser.lang)
     i18n.changeLanguage(this.state.loggedUser.lang)
 
-    document.addEventListener('appCustomEvent', this.customEventReducer)
+    document.addEventListener(CUSTOM_EVENT.APP_CUSTOM_EVENT, this.customEventReducer)
   }
 
   customEventReducer = ({ detail: { type, data } }) => { // action: { type: '', data: {} }
     switch (type) {
-      case 'workspace_advanced_showApp':
+      case CUSTOM_EVENT.SHOW_APP('workspace_advanced'):
         console.log('%c<WorkspaceAdvanced> Custom event', 'color: #28a745', type, data)
         this.setState({isVisible: true})
         this.loadContent()
         break
-      case 'workspace_advanced_hideApp':
+      case CUSTOM_EVENT.HIDE_APP('workspace_advanced'):
         console.log('%c<WorkspaceAdvanced> Custom event', 'color: #28a745', type, data)
         this.setState({isVisible: false})
         break
-      case 'workspace_advanced_reloadContent':
+      case CUSTOM_EVENT.RELOAD_CONTENT('workspace_advanced'):
```

Comment:
prefer the usage of `stage.config.slug` instead of hard coded 'workspace_advanced'